### PR TITLE
Support ko_KR and Fixed test code.

### PR DESCRIPTION
**Pull request by GTNH-Korean-Translation**

### CODE REVIEW

File: .vscode/settings.json
Comment (MuXiu1997) [inline .vscode/settings.json:1]: no `.vscode`

File: src/gtnh_translation_compare/settings/__init__.py
Comment (MuXiu1997) [inline src/gtnh_translation_compare/settings/__init__.py:1]: do not modify this the default value in the code, use environment variables to override

File: tests/filetypes/test_filetype_lang.py [hunk 1]
```diff
@@ -79,19 +95,25 @@ def test__get_properties(
...
 
 
 def test_get_zh_cn_relpath(
-    en_us_filetype_lang: FiletypeLang, zh_cn_filetype_lang: FiletypeLang, ja_jp_filetype_lang: FiletypeLang
+    en_us_filetype_lang: FiletypeLang, zh_cn_filetype_lang: FiletypeLang, ja_jp_filetype_lang: FiletypeLang, ko_kr_filetype_lang: FiletypeLang
 ) -> None:
-    assert en_us_filetype_lang.get_target_language_relpath(Language.zh_CN) == ZH_CN_RELPATH
-    assert zh_cn_filetype_lang.get_target_language_relpath(Language.ja_JP) == JA_JP_RELPATH
-    assert ja_jp_filetype_lang.get_target_language_relpath(Language.zh_CN) == ZH_CN_RELPATH
+    assert en_us_filetype_lang.get_target_language_relpath(Language.en_US) == EN_US_RELPATH
```
Comment (MuXiu1997) [inline tests/filetypes/test_filetype_lang.py:116]: The purpose of this part of the code may have been misunderstood by you

File: tests/filetypes/test_filetype_gt_lang.py [hunk 2]
```diff
@@ -102,23 +128,30 @@ def test__get_properties(
...
 def test_get_zh_cn_relpath(
     en_us_filetype_gt_lang: FiletypeGTLang,
     zh_cn_filetype_gt_lang: FiletypeGTLang,
     ja_jp_filetype_gt_lang: FiletypeGTLang,
+    ko_kr_filetype_gt_lang: FiletypeGTLang,
 ) -> None:
-    assert en_us_filetype_gt_lang.get_target_language_relpath(Language.zh_CN) == ZH_CN_RELPATH
-    assert zh_cn_filetype_gt_lang.get_target_language_relpath(Language.ja_JP) == JA_JP_RELPATH
-    assert ja_jp_filetype_gt_lang.get_target_language_relpath(Language.zh_CN) == ZH_CN_RELPATH
+    assert en_us_filetype_gt_lang.get_target_language_relpath(Language.en_US) == EN_US_RELPATH
```
Comment (MuXiu1997) [inline tests/filetypes/test_filetype_gt_lang.py:154]: The purpose of this part of the code may have been misunderstood by you

**Comment by MuXiu1997** [comment]
@GTNH-Korean-Translation

**Comment by MuXiu1997** [comment]
@GTNH-Korean-Translation 
I have merged the code and made modifications. You can use this script in GitHub Actions. Please use environment variables to override 'TARGET_LANG'

**Comment by GTNH-Korean-Translation** [comment]
thanks

**Pull request merged**